src/main/java/com/gotocompany/depot/maxcompute/converter/type/PrimitiveTypeInfoConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.gotocompany.depot.maxcompute.converter.type;
+
+import com.aliyun.odps.type.TypeInfo;
+import com.aliyun.odps.type.TypeInfoFactory;
+import com.google.protobuf.Descriptors;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PrimitiveTypeInfoConverter implements TypeInfoConverter {
+
+    private static final Map<Descriptors.FieldDescriptor.Type, TypeInfo> PROTO_TYPE_MAP;
+    static {
+        PROTO_TYPE_MAP = new HashMap<>();
```

Comment:
Additionally, to make it concise and readable and addressing immutability, please do apply the below changes

```
import static com.google.protobuf.Descriptors.FieldDescriptor.Type.*;

public class PrimitiveTypeInfoConverter implements TypeInfoConverter {

    private static final Map<Descriptors.FieldDescriptor.Type, TypeInfo> PROTO_TYPE_MAP;

    static {
        PROTO_TYPE_MAP = ImmutableMap.<Descriptors.FieldDescriptor.Type, TypeInfo>builder()
                .put(BYTES, TypeInfoFactory.BINARY)
                .put(STRING, TypeInfoFactory.STRING)
                .put(ENUM, TypeInfoFactory.STRING)
                .put(DOUBLE, TypeInfoFactory.DOUBLE)
                .put(FLOAT, TypeInfoFactory.FLOAT)
                .put(BOOL, TypeInfoFactory.BOOLEAN)
                .put(INT64, TypeInfoFactory.BIGINT)
                .put(UINT64, TypeInfoFactory.BIGINT)
                .put(INT32, TypeInfoFactory.INT)
                .put(UINT32, TypeInfoFactory.INT)
                .put(FIXED64, TypeInfoFactory.BIGINT)
                .put(FIXED32, TypeInfoFactory.INT)
                .put(SFIXED32, TypeInfoFactory.INT)
                .put(SFIXED64, TypeInfoFactory.BIGINT)
                .put(SINT32, TypeInfoFactory.INT)
                .put(SINT64, TypeInfoFactory.BIGINT)
                .build();
    }
```

I think this is more easily readable.